Cython/Compiler/Nodes.py

```diff
@@ -1934,11 +1934,16 @@ def generate_function_definitions(self, env, code):
                 code.putln(";")
             code.put(cenv.scope_class.type.declaration_code(Naming.outer_scope_cname))
             code.putln(";")
-        self.generate_argument_declarations(lenv, code)
+        tempvardecl_code = code.insertion_point()
+        tempvardecl_code.putln("")
```

Comment:
You're not using this only for temp variables. Let's rename it to `declaration_code`. And let's try to avoid sequences of empty lines in the C code in simple cases. I don't think we need empty lines here at all.